services/packagist/packagist-downloads.service.js

```diff
@@ -101,7 +101,7 @@ module.exports = class PackagistDownloads extends BasePackagistService {
   async handle({ interval, user, repo }, { server }) {
     const {
       package: { downloads },
-    } = await this.fetch({ user, repo, schema, server })
+    } = await this.fetchByJsonAPI({ user, repo, schema, server })
```

Comment:
It may also be helpful if we include a note in the documentation for the badges like this one that are using the JSON API that the data may be delayed by 12 hours.

Reply:
Did you refer to https://shields.io/category/downloads when you mentioned documentation?

I am not quite sure about the correct way to present information about possible delay.
Right now, badges are described just with short name with optional data within brackets.
Maybe `cached` keyword could be introduced (example attached).
This way, label would remain compact.
On the other hand, I am not convinced if this `cache` keyword would be understandable by users.

Maybe you already have another idea?

<img width="445" alt="Screenshot 2019-10-28 at 18 40 03" src="https://user-images.githubusercontent.com/4865322/67703524-ddf82480-f9b3-11e9-9b63-229007fd6804.png">

Reply:
Apologies for the confusion. I was referring to the documentation which appears to users within the modal window after clicking one of the sample listing which shows up above the parameter fields:

![image](https://user-images.githubusercontent.com/13042488/67706313-868f8f80-f986-11e9-98ef-0282a6067767.png)

We've got this on a few badges where it's helpful to convey additional info to users, for example:

Jenkins Coverage
![image](https://user-images.githubusercontent.com/13042488/67706419-b9398800-f986-11e9-869d-abf6e085c6f9.png)



Discord
![image](https://user-images.githubusercontent.com/13042488/67706472-d5d5c000-f986-11e9-8c6b-3b39803cb60a.png)


This is done by defining the desired documentation text within the service class, and attaching it to the example listing objects using the `documentation` key

https://github.com/badges/shields/blob/7e0976cd8cc65f012ee86f962aad08ce3aa16cb9/services/jenkins/jenkins-coverage.service.js#L82-L91

https://github.com/badges/shields/blob/7e0976cd8cc65f012ee86f962aad08ce3aa16cb9/services/jenkins/jenkins-coverage.service.js#L107-L122

Reply:
Thank you for complex explanation. I've totally missed how documentation property is working.

I've submitted change with additional documentation for json api delay.